src/signature.c

```diff
@@ -241,43 +242,45 @@ bool signature_verify(const char *file, const char *sig_file, bool print_errors)
 	/* get the signature */
 	sig_fd = open(sig_file, O_RDONLY);
 	if (sig_fd == -1) {
-		string_or_die(&errorstr, "Failed open %s: %s\n", sig_file, strerror(errno));
+		string_or_die(&errorstr, "Failed open %s: %s", sig_file, strerror(errno));
```

Comment:
Failed to open

Reply:
Well that typo in the sentence was not part of my commit, but sure, I can fix it.